.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: AI auto review
         id: ai-review
         uses: ./
-        with:
+        env:
```

Comment:
Is this intentional? `with` and `env` are not interchangeable. Environment variables do not support line breaks, so the content in `USER_PROMPT` defined below this key will change in a destructive manner by the time it reaches the action.

Reply:
This is discussed in the call. The following info is for users who might be reading it.

For actions with docker, always set inputs as env, this causes mangling. So we are doing a hack by directly reading the `ci.js.yml`   file.